### ISSUE TITLE
`th.eb` is never introduced in tutorial

### ISSUE THREAD

**Issue by ddetone** [facebookresearch/theseus#176]
## ❓ Questions and Help

I am going through the tutorial an noticed that `th.eb` is never introduced. Looking at the code it seems that `eb` stands for `embodied`? That seems odd that a core cost function is under than namespace

<img width="641" alt="image" src="https://user-images.githubusercontent.com/4764838/165122023-f43f0a05-ec7f-4a70-bb8a-e40ed5707d1d.png">

https://github.com/facebookresearch/theseus/blob/main/tutorials/00_introduction.ipynb

**Comment by luisenp** [comment]
Yes, that's a good observation. I also agree that `VariableDifference` (and maybe `Between`?) should move up to core folder. @mhmukadam  and I were also discussing renaming this function (see #174), so we could do these two changes together.
